DESCRIPTION

```diff
@@ -75,13 +75,12 @@ RdMacros:
 Config/Needs/verdepcheck: rstudio/shiny, insightsengineering/teal.data,
     insightsengineering/teal.slice, mllg/checkmate,
     HenrikBengtsson/future, jeroen/jsonlite, r-lib/lifecycle,
-    daroczig/logger, tidyverse/magrittr, rstudio/promises, rstudio/renv,
-    r-lib/rlang, daattali/shinyjs, insightsengineering/teal.logger,
+    daroczig/logger, rstudio/promises, rstudio/renv,
+    r-lib/rlang, daattali/shinyjs, insightsengineering/teal.code, insightsengineering/teal.logger,
     insightsengineering/teal.reporter, insightsengineering/teal.widgets,
     rstudio/bslib, yihui/knitr, bioc::MultiAssayExperiment, r-lib/R6,
-    rstudio/rmarkdown, rstudio/shinyvalidate,
-    insightsengineering/teal.code, r-lib/testthat, r-lib/withr,
-    yaml=vubiostat/r-yaml
+    rstudio/rmarkdown, tidyverse/rvest, rstudio/shinytest2, rstudio/shinyvalidate,
+    r-lib/testthat, r-lib/withr, yaml=vubiostat/r-yaml
```

Comment:
FYI:
- removed `magrittr`
- moved `teal.code` to align with dependencies
- added `rvest`
- added `shinytest2`